### ISSUE TITLE
NTR: 18-hydroxycorticosterone / aldosterone ratio high (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8357]
New term request 18-hydroxycorticosterone / aldosterone ratio high (elthat_130803081907):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000515
- label: Elevated 18-hydroxycorticosterone to aldosterone ratio
- parents: [Abnormal circulating corticosterone level (HP:0012112)](https://hpo.jax.org/app/browse/term/HP:0012112)
- pmid: PMID:2981240
- def: The relative concentration in the blood circulation of 18-hydroxycorticosterone compared to that of aldosterone is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8357
- synonyms:
